app/models/promotion.rb

```diff
@@ -0,0 +1,5 @@
+class Promotion < ApplicationRecord
+  validates :amount, :value, presence: true
+  validates :value, length: { maximum: 255, too_long: '%<count>s characters is the maximum allowed' }
```

Comment:
how are you going to get the value to calculate the discount? 🤔 You could have value as integer and information as a string to what is the discount. Perhaps the client would like to have a discount on tickets you are buying right now e.g. when user selected 2 seats we could have a promotion for 10% discount if user picks another seat

Reply:
I have this information in amount, but yeah I can change it to value and information, it looks better